changelogutils/reader_test.go

```diff
@@ -41,8 +40,22 @@ var _ = Describe("ReaderTest", func() {
 			file = changelogutils.ChangelogFile{
 				Entries: []*changelogutils.ChangelogEntry{&entry},
 			}
+			log  = "1.yaml"
+			repo = "testrepo"
+			sha  = "9065a9a84e286ea7f067f4fc240944b0a4d4c82a"
```

Comment:
Here, I'm letting the testrepo flavor continue to exist, although I'd be inclined to just use the public repositories entirely for the normal stuff and reserve if for the special cases.